apps/launcher/graphicspage.cpp

```diff
@@ -47,7 +47,10 @@ Launcher::GraphicsPage::GraphicsPage(QWidget *parent)
     connect(screenComboBox, SIGNAL(currentIndexChanged(int)), this, SLOT(screenChanged(int)));
     connect(framerateLimitCheckBox, SIGNAL(toggled(bool)), this, SLOT(slotFramerateLimitToggled(bool)));
     connect(shadowDistanceCheckBox, SIGNAL(toggled(bool)), this, SLOT(slotShadowDistLimitToggled(bool)));
+}
 
+void Launcher::GraphicsPage::connectAntiAliasingChanged(const QObject* receiver, const char* slot) {
+    connect(antiAliasingComboBox, SIGNAL(currentIndexChanged(int)), receiver, slot);
```

Comment:
Probably it would be better to unify the code - we do not use a separate function to connect every checkbox.

Reply:
I was torn on what to do in this case, because I need to connect elements between pages. Do you know of any existing connections like this I can base mine off of?
This is the combo box for the different levels of anti-aliasing. `advancedpage` contains the `antialiasAlphaTestCheckBox` if you'd like to take a look!

Reply:
Probably it is not a good idea to add dependencies between settings from different tabs. Also OpenMW itself just does not use `antialias alpha test` when there is no antialiasing enabled, so maybe an additional logic in the launcher is not needed.

Reply:
Ok, I can remove the dependency. However, if I do it will be possible to have alpha test enabled and AA disabled in the settings file and I don't think I can override if this is the case. If that still sounds OK, I can go ahead and push a new commit. I did see the engine side checks for antialiasing to be enabled, so it doesn't sound like too big of a deal.

Reply:
Has this been addressed?

Reply:
I did end up removing the connection between pages. As I mentioned above, it is possible for users to have the alpha test enabled when AA is off because of that. I verified that the AA setting is checked on the engine side whenever it's loaded.